hwdb.d/60-keyboard.hwdb

```diff
@@ -523,6 +523,14 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnGeminiDevices:pnNC14V1006:*
 # Genius
 ###########################################################
 
+###########################################################
+# Haier
+###########################################################
+
+# 7G-Series
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHaierComputer:pn7G-Series:*
+ KEYBOARD_KEY_91=wlan
+
```

Comment:
Then, please move the new "Haier" section above "Huawei" section, to make the sections sorted alphabetically.

Reply:
> Then, please move the new "Haier" section above "Huawei" section, to make the sections sorted alphabetically.

I think it should be placed above Hewlett-Packard too, to be alphabetically sorted.

Reply:
Yeah. You are right.